### ISSUE TITLE
Degree symbol 

### ISSUE THREAD

**Issue by mike-2545** [yozik04/nextion#23]
I cannot get a degree symbol to pass to the Nextion from Raspberry Pi Zero Python
```
output = Fahrenheit ,(degree_sign,'utf-8')
await client.set('InTemp.txt', str(output))
```
UnicodeEncodeError: 'ascii' codec can't encode character '\xb0' in position 19: ordinal not in range(128)

Mike

**Comment by yozik04** [comment]
Try to pass it in nextion editor. Does it work?

**Comment by mike-2545** [comment]
Yes it works in the Nextion editor, for python I added client.encoding = 'latin-1' and it works for text boxes but the scrolling text box does not .

 

Mike

**Comment by yozik04** [comment]
> it works for text boxes but the scrolling text box does not.

I need code to reproduce. I have not tried to do anything with scrolling text boxes.
You'd better capture what Nextion Editor sends and what my lib sends so we can see the difference.
